dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/mapper/RequestToSearchParamsMapper.java

```diff
@@ -0,0 +1,266 @@
+ * ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.util.Date;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.hisp.dhis.category.CategoryOptionCombo;
+import org.hisp.dhis.common.AssignedUserSelectionMode;
+import org.hisp.dhis.common.DimensionalObject;
+import org.hisp.dhis.common.IdSchemes;
+import org.hisp.dhis.common.IllegalQueryException;
+import org.hisp.dhis.common.OrganisationUnitSelectionMode;
+import org.hisp.dhis.common.QueryFilter;
+import org.hisp.dhis.common.QueryItem;
+import org.hisp.dhis.common.QueryOperator;
+import org.hisp.dhis.dataelement.DataElement;
+import org.hisp.dhis.dataelement.DataElementService;
+import org.hisp.dhis.dxf2.events.event.EventSearchParams;
+import org.hisp.dhis.event.EventStatus;
+import org.hisp.dhis.organisationunit.OrganisationUnit;
+import org.hisp.dhis.organisationunit.OrganisationUnitService;
+import org.hisp.dhis.program.Program;
+import org.hisp.dhis.program.ProgramService;
+import org.hisp.dhis.program.ProgramStage;
+import org.hisp.dhis.program.ProgramStageService;
+import org.hisp.dhis.program.ProgramStatus;
+import org.hisp.dhis.query.Order;
+import org.hisp.dhis.security.acl.AclService;
+import org.hisp.dhis.trackedentity.TrackedEntityInstance;
+import org.hisp.dhis.trackedentity.TrackedEntityInstanceService;
+import org.hisp.dhis.user.CurrentUserService;
+import org.hisp.dhis.user.User;
+import org.hisp.dhis.user.UserCredentials;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+
+/**
+ * @author Luciano Fiandesio
+ */
+@Component
+public class RequestToSearchParamsMapper
+{
+    private final CurrentUserService currentUserService;
+
+    private final ProgramService programService;
+
+    private final OrganisationUnitService organisationUnitService;
+
+    private final ProgramStageService programStageService;
+
+    private final AclService aclService;
+
+    private final TrackedEntityInstanceService entityInstanceService;
+
+    private final DataElementService dataElementService;
+
+    public RequestToSearchParamsMapper( CurrentUserService currentUserService, ProgramService programService,
+        OrganisationUnitService organisationUnitService, ProgramStageService programStageService, AclService aclService,
+        TrackedEntityInstanceService entityInstanceService, DataElementService dataElementService )
+    {
+        checkNotNull( currentUserService );
+        checkNotNull( programService );
+        checkNotNull( organisationUnitService );
+        checkNotNull( programStageService );
+        checkNotNull( aclService );
+        checkNotNull( entityInstanceService );
+        checkNotNull( dataElementService );
```

Comment:
newline

Reply:
I guess `newline` means add a new line between `checkNotNull` and the dependency setting. Fixed